stage_1/stage_1.ipynb

```diff
@@ -493,7 +493,7 @@
  ],
```

Comment:
seems like there's another file written by Wataru has the same name as this file? Did you modify the file?